src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family for the case when the 'Roboto' font is not available on the user's system. For example, 'Roboto, Arial, sans-serif'.